packages/reports/package.json

```diff
@@ -82,7 +82,7 @@
     "ember-qunit-assert-helpers": "^0.2.0",
     "ember-resize": "0.0.17",
     "ember-resolver": "^4.0.0",
-    "ember-sortable": "^1.12.1",
+    "ember-sortable": "1.12.3",
```

Comment:
This package is in more than just reports and core.  Should it be pinned in all the subpackages as well?